.gitignore

```diff
@@ -6,12 +6,11 @@ lib
 build
 docs
 result
-conductor-*.toml
-keystore-*.key
-AGENTID
-conductor.log
-sim2h.log
-holochain-conductor/storage
 hc-conductor.log
 hc-lair.log
-tests/tmp
+script/install-bundles/.hc*
```

Comment:
I was a little confused by the name `script` for the folder.  This seems like it's all about testing, so maybe this should be the name somehow?  Not a big deal a but a bit confusing at first.